### PR TITLE
Update Uninstall Links.

### PR DESCRIPTION
**Pull request by Shubham82**
Signed-off-by: Shubham82 <shubham.kuchhal@india.nec.com>

Update the links for deleting the crds during uninstall of keda1.5.

### CODE REVIEW

File: content/docs/1.5/deploy.md [hunk 1]
```diff
@@ -51,16 +51,16 @@ If you want to remove KEDA from a cluster you can run one of the following:
 
 ```sh
 helm uninstall -n keda keda
-kubectl delete -f https://raw.githubusercontent.com/kedacore/keda/master/deploy/crds/keda.k8s.io_scaledobjects_crd.yaml
-kubectl delete -f https://raw.githubusercontent.com/kedacore/keda/master/deploy/crds/keda.k8s.io_triggerauthentications_crd.yaml
+kubectl delete -f https://raw.githubusercontent.com/kedacore/keda/v1/deploy/crds/keda.k8s.io_scaledobjects_crd.yaml
```
Comment (tomkerkhove) [inline content/docs/1.5/deploy.md:54]:
```suggestion
kubectl delete -f https://raw.githubusercontent.com/kedacore/keda/v1.5.0/deploy/crds/keda.k8s.io_scaledobjects_crd.yaml
```

File: content/docs/1.5/deploy.md [hunk 2]
```diff
@@ -51,16 +51,16 @@ If you want to remove KEDA from a cluster you can run one of the following:
...
+kubectl delete -f https://raw.githubusercontent.com/kedacore/keda/v1/deploy/crds/keda.k8s.io_triggerauthentications_crd.yaml
```
Comment (tomkerkhove) [inline content/docs/1.5/deploy.md:55]:
```suggestion
kubectl delete -f https://raw.githubusercontent.com/kedacore/keda/v1.5.0/deploy/crds/keda.k8s.io_triggerauthentications_crd.yaml
```

File: content/docs/1.5/deploy.md [hunk 3]
```diff
@@ -51,16 +51,16 @@ If you want to remove KEDA from a cluster you can run one of the following:
...
 ```
 
 **Using Helm 2**
 
 ```sh
 helm delete --purge keda
-kubectl delete -f https://raw.githubusercontent.com/kedacore/keda/master/deploy/crds/keda.k8s.io_scaledobjects_crd.yaml
-kubectl delete -f https://raw.githubusercontent.com/kedacore/keda/master/deploy/crds/keda.k8s.io_triggerauthentications_crd.yaml
+kubectl delete -f https://raw.githubusercontent.com/kedacore/keda/v1/deploy/crds/keda.k8s.io_scaledobjects_crd.yaml
```
Comment (tomkerkhove) [inline content/docs/1.5/deploy.md:62]:
```suggestion
kubectl delete -f https://raw.githubusercontent.com/kedacore/keda/v1.5.0/deploy/crds/keda.k8s.io_scaledobjects_crd.yaml
```

File: content/docs/1.5/deploy.md [hunk 4]
```diff
@@ -51,16 +51,16 @@ If you want to remove KEDA from a cluster you can run one of the following:
...
+kubectl delete -f https://raw.githubusercontent.com/kedacore/keda/v1/deploy/crds/keda.k8s.io_triggerauthentications_crd.yaml
```
Comment (tomkerkhove) [inline content/docs/1.5/deploy.md:63]:
```suggestion
kubectl delete -f https://raw.githubusercontent.com/kedacore/keda/v1.5.0/deploy/crds/keda.k8s.io_triggerauthentications_crd.yaml
```

**Review by tomkerkhove — CHANGES_REQUESTED** [review]
Good catch! Added some suggestions to make it even a bit more precise

**Comment by Shubham82** [comment]
Thanks @tomkerkhove  for your suggestion.
i will do the changes  as per you mention.

**Comment by Shubham82** [comment]
Hi @tomkerkhove  
I have addressed the changes PTAL.
Thanks

**Review by tomkerkhove — APPROVED** [review]

**Pull request merged**